Exercise_2.py

```diff
@@ -1,15 +1,52 @@
-
+#Time Complexity : O(n)
+#Space Complexity : O(n)
+#Did this code successfully run on Leetcode :
+#Any problem you faced while coding this :
 class Node:
     def __init__(self, data):
        self.data = data
        self.next = None
  
 class Stack:
     def __init__(self):
-        
+        self.head = None
+        self.second_last = None
+        self.last = None
     def push(self, data):
```

Comment:
You need to just add/remove element at head of linked list. That will be act as a stack .

Reply:
Which will be in O(1)

Reply:
Oh yes. Slipped out of my mind. Will note it